### PR TITLE
Bugs #324, 325 fixed, scroll on source tables panel moved to left

### PR DESCRIPTION
**Pull request by MariaDolotova**

### CODE REVIEW

File: UI/src/app/components/comfy-search-by-name/comfy-search-by-name.component.ts [hunk 1]
```diff
@@ -60,14 +60,18 @@ export class ComfySearchByNameComponent extends BaseComponent
   }
 
   search(textCriteria: string) {
+    if(textCriteria==''){
```
Comment (etokareva) [inline UI/src/app/components/comfy-search-by-name/comfy-search-by-name.component.ts:63]: Please format this

File: UI/src/app/components/comfy/comfy.component.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h3 class="filter-label">Link tables</h3>
         (reset)="filterByNameReset('source', $event)"
       ></app-comfy-search-by-name>
       <h4 class="page-column-label">Source tables</h4>
-      <div class="scrolled-items">
+      <div class="scrolled-items" style="direction:rtl;">
```
Comment (etokareva) [inline UI/src/app/components/comfy/comfy.component.html:33]: add this style into scrolled-items

Comment (etokareva) [reply UI/src/app/components/comfy/comfy.component.html:33]: Or create and use new class:
`.scrolled-items-rtl {
   @extend .scrolled-items;
   direction: rtl;
}`

File: UI/src/app/components/comfy-search-by-name/comfy-search-by-name.component.ts (hunk 1 above)
Comment (MariaDolotova) [reply UI/src/app/components/comfy-search-by-name/comfy-search-by-name.component.ts]: Fixed in commit #6afb96d

File: UI/src/app/components/comfy/comfy.component.html (hunk 2 above)
Comment (MariaDolotova) [reply UI/src/app/components/comfy/comfy.component.html]: Fixed in commit #6afb96d

**Pull request merged**